webpack.config.js

```diff
@@ -1,100 +0,0 @@
-const path = require('path');
-const StyleLintPlugin = require('stylelint-webpack-plugin');
-const VisualizerPlugin = require('webpack-visualizer-plugin');
-
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 2900000;
-const MAX_PRODUCTION_OUTPUT_SIZE = 335000;
-
-module.exports = (env, argv) => ({
-  devtool: argv.mode === 'production'
-    ? false
-    : 'eval-cheap-module-source-map',
-  entry: {
-    lib: [
-      path.join(__dirname, 'src/lib/index.js'),
-    ],
-  },
-  externals: {
```

Comment:
Change of those might broke dependent project. Why it has been changed? I remember that it took to configure this properly to make RUI working ...

Reply:
I added it back